sdks/golang/auth/auth.go

```diff
@@ -0,0 +1,37 @@
+package auth
+
+import (
+	"github.com/microsoft/kiota-abstractions-go/authentication"
+	"sync"
+)
+
+type EkaJwtProvider interface {
```

Comment:
@vishal24tuniki This is the only interface that you should implement.

Reply:
Because the Auth Token is not taken as a Bearer Standard, so we had to extend the functionality using a custom implementation,